vro-types/o11n-plugin-aria/index.d.ts

```diff
@@ -663,15 +676,355 @@ declare class VraNetworkInterfaceSpecification {
 	 * @param key
 	 * @param customPropertiesItem
 	 */
-	putCustomPropertiesItem(key: string, customPropertiesItem: string): VraNetworkInterfaceSpecification;
+	putCustomPropertiesItem(
+		key: string,
+		customPropertiesItem: string
+	): VraNetworkInterfaceSpecification;
 
 	/**
-   * @param securityGroupIdsItem
-   */
-	addSecurityGroupIdsItem(securityGroupIdsItem: string): VraNetworkInterfaceSpecification;
+	 * @param securityGroupIdsItem
+	 */
+	addSecurityGroupIdsItem(
+		securityGroupIdsItem: string
+	): VraNetworkInterfaceSpecification;
 
 	/**
-   * @param addressesItem
-   */
+	 * @param addressesItem
+	 */
 	addAddressesItem(addressesItem: string): VraNetworkInterfaceSpecification;
 }
+
+declare class VraCloudAccount {
+	owner: string;
+	linksExtension: string;
+	cloudAccountType: string;
+	description: string;
+	enabledRegionIdsExtension: string;
+	orgId: string;
+	tags: Object[];
+	createdAt: string;
+	enabledRegions: string;
```

Comment:
I think this one should be array of objects:
![image](https://github.com/user-attachments/assets/bf229ba0-212d-419c-b405-cc47fd7acdf7)

Reply:
Could you double check, I think by mistake you changed the enabledRegionIdsExtension instead, which should remain a string

Reply:
Yuo're right. My bad. Fixed.

Reply:
Awesome, thanks!